index.html

```diff
@@ -225,11 +225,11 @@ <h2 class="formTitle">I'm always interested in hearing about new projects, so if
   <section class="form2">
     <div id="marginTop">
       <ul class="iconsBot">
-        <li><a href="https://github.com/Zeraltz"><img src="./img/social/dark-icons/Github.png" alt=""></a></li>
-        <li><a href="https://www.linkedin.com/in/mauricio-cantillo-moreno-374576184/"><img src="./img/social/dark-icons/Linkedin.png" alt=""></a></li>
-        <li><a href="https://www.angellist.com/"><img src="./img/social/dark-icons/Angelist.png" alt=""></a></li>
-        <li><a href="https://www.twitter.com/MauroCantillo_"><img src="./img/social/dark-icons/Twitter.png" alt=""></a></li>
-        <li><a href="https://www.medium.com"><img src="./img/social/dark-icons/Medium.png" alt=""></a></li>
+        <li><a href="https://github.com/Zeraltz"><img src="./img/social/dark-icons/Github.png" alt="link-text"></a></li>
+        <li><a href="https://www.linkedin.com/in/mauricio-cantillo-moreno-374576184/"><img src="./img/social/dark-icons/Linkedin.png" alt="link-text"></a></li>
+        <li><a href="https://www.angellist.com/"><img src="./img/social/dark-icons/Angelist.png" alt="link-text"></a></li>
+        <li><a href="https://www.twitter.com/MauroCantillo_"><img src="./img/social/dark-icons/Twitter.png" alt="link-text"></a></li>
+        <li><a href="https://www.medium.com"><img src="./img/social/dark-icons/Medium.png" alt="link-text"></a></li>
```

Comment:
- [ ] Kindly change the `alt` attribute with an appropriate alternative text, that's useful for interacting or understanding.
Like for the GitHub icon change it to `GitHub account` for example.

- [ ] Kindly change all your alt attributes that don't have descriptive text.

Reply:
@keroloussamy I did the report with all the changes I did, please let me know what do you mean by this: 
- "On the Pull request description, please write a detailed accessibility report about all the checks you made on your website. You should have something similar to this"
That's exactly what I did, 
The other changes Ill do them right away

Reply:
Hello @Zeraltz 
Just put the points that you have checked and if there is something that there's no issue with it write next to it `no issue found` as an example `Page Title - no issue found`.
And if there's something that you upgraded write next to it what you did. Like `Image text alternatives - Fixed link images missing alternate text`